packages/dds/counter/README.md

```diff
@@ -1,94 +1,173 @@
-# @fluidframework/cell
+# @fluidframework/counter
 
-The `SharedCell` Distributed Data Structure (DDS) stores a single, shared value that can be edited or deleted.
-
-<!-- AUTO-GENERATED-CONTENT:START (README_INSTALLATION_SECTION:includeHeading=TRUE) -->
+<!-- AUTO-GENERATED-CONTENT:START (README_DEPENDENCY_GUIDELINES_SECTION:includeHeading=TRUE) -->
 
 <!-- prettier-ignore-start -->
 <!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
 
-## Installation
+## Using Fluid Framework libraries
 
-To get started, install the package by running the following command:
+When taking a dependency on a Fluid Framework library, we recommend using a `^` (caret) version range, such as `^1.3.4`.
+While Fluid Framework libraries may use different ranges with interdependencies between other Fluid Framework libraries,
+library consumers should always prefer `^`.
 
-```bash
-npm i @fluidframework/counter
-```
+Note that when depending on a library version of the form `2.0.0-internal.x.y.z`, called the Fluid internal version scheme,
+you must use a `>= <` dependency range (such as `>=2.0.0-internal.x.y.z <2.0.0-internal.w.0.0` where `w` is `x+1`).
+Standard `^` and `~` ranges will not work as expected.
+See the [@fluid-tools/version-tools](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/version-tools/README.md)
+package for more information including tools to convert between version schemes.
 
```

Comment:
If you re-run `build:readme` from the root, these sections about internal version numbers will go away. Tyler removed these from the markdown-magic templates after the website 2.0 PR went in.

```suggestion
```